src/main/java/gg/projecteden/nexus/features/chat/alerts/AlertsCommand.java

```diff
@@ -104,6 +115,23 @@ void partialMatching(boolean partialMatching, String highlight) {
 		Tasks.wait(2, () -> PlayerUtils.runCommand(player(), "alerts edit"));
 	}
 
+	@Path("negated <truse|false> [highlight...]")
+	void negated(boolean negated, String highlight) {
+		Optional<Alerts.Highlight> match = alerts.get(highlight);
+		if (!match.isPresent())
+			error("I could not find that alert in your alerts list");
+		boolean partialMatching = match.get().isPartialMatching(); // Is this good?
```

Comment:
We could remove this and the 2 lines below deleting and adding the highlight back by just using `match.setNegated(!match.isNegated)` and then saving